daemon/src/seed.rs

```diff
@@ -97,6 +99,12 @@ impl Seed for RandomSeed {
     }
 }
 
+impl Debug for RandomSeed {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("RandomSeed").field(&"...").finish()
+    }
+}
```

Comment:
❓ Should we just skip logging the seed? It's almost certainly not useful information and at the same time logging it can be a security issue.

Reply:
I think I added it to be able to derive Debug for something else, without having to reimplement everything in that debug impl. This debug doesn't actually print the seed - it always prints `RandomSeed(...)`

Reply:
Jesus, I'm blind. Sorry.